CONSTITUTION.md

```diff
@@ -182,9 +183,9 @@ In the absence of both the President and the Vice-President, the Committee membe
 
 ### 7.5. Advertising Meetings
 
-- 7.5.1. Notice of the Annual and Ordinary General Meetings including time, place and business to be transacted shall be given to all financial members of the Society at least seven (7) days before the date of any such meeting.
+- 7.5.1. Notice of the Annual and Ordinary General Meetings including time and place shall be given to all financial members of the Society at least fourteen (14) days before the date of any such meeting, with notice of business to be transacted given at least seven (7) days ahead of the meeting.
 
-- 7.5.2. Notice of all Special General Meetings shall be given to all financial members of the Society at least seven (7) days before the date of such meeting.
+- 7.5.2. Notice of all Special General Meetings shall be given to all financial members of the Society at least fourteen (14) days before the date of such meeting.
```

Comment:
Why more notice for an SGM?

Reply:
I'm going through and updating the amendments with rationales, thanks for pointing that out in the Discord - very easy to get all caught up and forget to explain it properly.

> Why more notice for an SGM?

ActivateUTS policy has SGM and AGM both at 14 day notices.

They haven't asked us to update it specifically, but it is confusing enough (requiring digging through their policies and procedures) that it would be good to update and make it simpler for future Execs.